mongodb-quickstart/src/main/java/org/acme/mongodb/CodecFruitResource.java

```diff
@@ -0,0 +1,33 @@
+package org.acme.mongodb;
+
+import org.bson.types.Code;
+
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/codec_fruits")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class CodecFruitResource {
```

Comment:
I'm not sure about the codec prefix. Any reason for this?

Reply:
I introduced this resource to complete the 3 approaches present in this QS, see also https://github.com/quarkusio/quarkus-quickstarts/pull/938#discussion_r711107489.